packages/gestalt/src/utils/positioningUtils.js

```diff
@@ -167,18 +172,27 @@ export function getPopoverDir({
     up = 0;
     down = 0;
   }
-  const spaces = [up, right, down, left];
 
-  // Identify best direction of available spaces
-  const max = Math.max(...spaces);
   // Chose the main direction for the popover based on available spaces & user preference
+  const spaces = [up, right, down, left];
+  const noAvailableSpaceConditionSpaces = [up, down];
+
   let popoverDir;
+
   if (idealDirection && spaces[DIR_INDEX_MAP[idealDirection]] > 0) {
     // user pref
     popoverDir = idealDirection;
   } else {
+    const noAvailableSpaceCondition = up <= 0 && right <= 0 && down <= 0 && left <= 0;
```

Comment:
Could you add some extra comments detailing this change?

Reply:
this code will go away when this one gets merged.>>  ScrollBoundaryContainer: fix positioning within when no available spa… https://github.com/pinterest/gestalt/pull/1421

Reply:
Ahh cool, nvm then!